src/components/Character.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Character = ({ image, name }) => (
+  <Wrapper>
+    <li className="character">
+      {image && <img src={image} alt={name} className="image" />}
+      {!image && <h2 className="not-found">Image not found</h2>}
+      <div className="txt">
+        <p>{name}</p>
+      </div>
+    </li>
+  </Wrapper>
+);
+
+Character.propTypes = {
+  image: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+};
+
+// <img src={image} alt={name} className="image"/>
```

Comment:
- [ ] it would be better to remove the the unused commented code to keep the file clean and pro

Reply:
Thanks for the review i will make the requested changes